keyboards/cutie_club/fidelity/readme.md

```diff
@@ -0,0 +1,26 @@
+# Fidelity
+
+![Fidelity](https://i.imgur.com/6xelfJe.jpeg)
```

Comment:
```suggestion
![Fidelity](https://i.imgur.com/6xelfJeh.jpeg)
```

Reply:
Apologies, I missed that, thank you for the correction \(★ω★)/